README.md

```diff
@@ -92,10 +92,10 @@ $message = new Message("MSH|^~\\&|1|\rPV1|1|O|^AAAA1^^^BB|", null, true); // Thi
 $pv1 = $message->getSegmentByIndex(1);
 $fields = $pv1->getField(3); // $fields is ['', 'AAAA1', '', '', 'BB']
 
-// Create/send message with segment-ending bar character (|) removed
-$message = new Message("MSH|^~\\&|1|\nABC|||xxx\n", ['SEGMENT_ENDING_BAR' => false]);
+// Create/send message with segment-ending field-separator character (default "|") removed
```

Comment:
Can you also add following line under the "important" section in Readme:
> `SEGMENT_ENDING_BAR` is deprecated and will be removed in a future release. Use `WITH_SEGMENT_ENDING_FIELD_SEPARATOR` instead.

Reply:
_Important_ wasn't a section yet, so I reworked it a little.